src/Line.js

```diff
@@ -46,6 +46,7 @@ class Line {
       datasets,
     };
     this.options = options;
+    this.filter = !options.disableEffect ? 'url(#xkcdify)' : null;
     this.svgEl = select(svg)
       .style('stroke-width', '3')
       .style('font-family', this.options.fontFamily || 'xkcd')
```

Comment:
Besides turning off filter, turning off the 'xkcd' font seems necessary too

Reply:
I'm not sure. Regarding the original issue, there are issues with chart readability, but the font is rather readable in my opinion. Moreover, there is already the option to change font family

Reply:
Make sense, this is not mentioned in the original issue and not a must have feature. We can do this later if people want it